File: src/main/resources/templates/src/app/model/pedido.ts
import { Cliente } from '../model/cliente';

export class Pedido {
  constructor(
    public id?: number,
    idCliente?: number | undefined,
    public data?: Date,
    public situacao?: boolean,
    public valorTotal?: number,
    public rua?: String,
    public numero?: number,
    public bairro?: string,
    public cidade?: String,
    public estado?: String,
    public cep?: String,
    public descontos?: number
  ) { }
}

File: src/main/java/com/app_pedidos/model/entity/Pedido.java
package com.app_pedidos.model.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import java.time.LocalDate;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class Pedido {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    LocalDate data = LocalDate.now();
    @ManyToOne
    @JoinColumn(name = "cliente_id")
    private Cliente cliente;
    private boolean situacao = true;
    private String rua;
    private String numero;
    private String bairro;
    private String cidade;
    @JoinColumn(nullable = true)
    private String estado;
    private String cep;
    private double valorTotal;
	   
    
    

}
